exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,15 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  let string = sentence;
+  let word = string.includes("code");
+  if ((string = word)) {
```

Comment:
In line 14 to note that here you used the "=" Assignment operator instead of the Comparison operator which would mean that instead of comparing you are making this statement equal the value of word.  so if for e.g string includes Code, word would be true, but then String would now be reassigned true, and it would read as "If String is true - Return "true". But if string did not include code. word would hold false but string would still be assigned false and it would read as "If string is false - Return true.